App.tsx

```diff
@@ -1,53 +1,54 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import { ActivityIndicator, View } from "react-native";
 import { NavigationContainer } from "@react-navigation/native";
 import { createNativeStackNavigator } from "@react-navigation/native-stack";
-import { OrganizationListView } from "./src/pages/organizations/OrganizationListView";
-import { ViewLayoutStructure } from "./src/components/ViewLayoutStructure";
-import { OrganizationDetailsView } from "./src/pages/organization-details/OrganizationDetailsView";
-import { EventDetailsView } from "./src/pages/event-details/EventDetailsView";
-import { SamplePage } from "./src/pages/SamplePage";
-import { SettingsView } from "./src/pages/settings/SettingsView";
 import { I18nextProvider } from "react-i18next";
 import i18next from "./src/localization/i18next";
-import { CalendarScreen } from "./src/pages/calendar/CalendarScreen";
-import { EventSearchScreen } from "./src/pages/event-search/EventSearchScreen";
+import { AuthenticationService } from "./src/services/AuthenticationService";
+import { MainStack } from "./src/navigationstacks/MainStack";
+import { AuthenticationStack } from "./src/navigationstacks/AuthenticationStack";
 
 const Stack = createNativeStackNavigator();
 
 export default function App() {
+  const [firstScreen, setFirstScreen] = useState("");
```

Comment:
wrzucil bym to w cos jakiegos hooka bo tez "Main" i "Login" nie bardzo jest dobrym sposobem na uzywanie roznych screenow